feature/bgp/policybase/otg_tests/chained_policies_test/chained_policies_test.go

```diff
@@ -383,18 +383,32 @@ func configureExportRoutingPolicy(t *testing.T, dut *ondatra.DUTDevice, operatio
 	if !deviations.DefaultImportExportPolicy(dut) {
 		policy.SetDefaultExportPolicy(oc.RoutingPolicy_DefaultPolicyType_REJECT_ROUTE)
 	}
```

Comment:
Sure @dplore , will update it.
I was re using existing deviation in script, will modify it and resubmit.

Reply:
@dplore  updated comment as you suggested. Thank you.

Reply:
@cprabha Would you also remove the existing deviation `DefaultImportExportPolicy` with `DefaultImportExportPolicyUnsupported`, otherwise the two deviations will unnecessarily cause some confusion?

Reply:
@self-maurya  I have updated other scripts which were using DefaultImportExportPolicy  and updated metadata files. Please check